release-common-lib/src/main/java/org/reactome/release/common/database/InstanceEditUtils.java

```diff
@@ -10,34 +10,37 @@
 import org.gk.schema.SchemaClass;
 import org.gk.util.GKApplicationUtilities;
 
+/**
+ * This class provides methods to create InstanceEdits as GKInstance objects
+ * @author sshorser
+ */
 public class InstanceEditUtils
 {
 	/**
-	 * Create an InstanceEdit.
-	 * 
-	 * @param personID
-	 *            - ID of the associated Person entity.
-	 * @param creatorName
-	 *            - The name of the thing that is creating this InstanceEdit.
-	 *            Typically, you would want to use the package and classname that
-	 *            uses <i>this</i> object, so it can be traced to the appropriate
-	 *            part of the program.
-	 * @return
-	 * @throws Exception 
+	 * Creates an InstanceEdit object associated with a specific person id and creator name
+	 *
+	 * @param dba MySQLAdaptor connecting to the database for which the InstanceEdit will be created and stored
+	 * @param personID ID of the associated Person instance in the database
+	 * @param creatorName The name of the thing that is creating this InstanceEdit.  Typically, you would want to use
+	 * the package and classname that uses <i>this</i> object, so it can be traced to the appropriate part of the
+	 * program.
+	 * @return InstanceEdit as a GKInstance object
+	 * @throws Exception Thrown if unable to create the GKInstance object representing an InstanceEdit
 	 */
-	public static GKInstance createInstanceEdit(MySQLAdaptor adaptor, long personID, String creatorName) throws Exception
+	public static GKInstance createInstanceEdit(MySQLAdaptor dba, long personID, String creatorName)
+		throws Exception
 	{
 		GKInstance instanceEdit = null;
 		try
 		{
-			instanceEdit = createDefaultIE(adaptor, personID, true, "Inserted by " + creatorName);
+			instanceEdit = createDefaultIE(dba, personID, true, "Inserted by " + creatorName);
```

Comment:
For cases where we are actually just modifying an instance, does it makes sense to use the phrase 'Inserted by'?

Reply:
I had an idea (I forget who I talked to about it, Joel or you?) about having a cache of InstanceEdits with more detailed descriptions, so that different steps could choose an InstanceEdit with a better Note.

This has not yet been implemented. :(

Reply:
There's also the method 
`public static GKInstance createDefaultIE(MySQLAdaptor dba, Long defaultPersonId, boolean needStore, String note)` 
to allow a generic note.

Reply:
> I had an idea (I forget who I talked to about it, Joel or you?) about having a cache of InstanceEdits with more detailed descriptions, so that different steps could choose an InstanceEdit with a better Note.
> 
> This has not yet been implemented. :(

Yes, I think Solomon and I talked about that.